app/scripts/modules/core/src/pipeline/config/stages/pipeline/PipelineParametersExecutionDetails.tsx

```diff
@@ -22,7 +22,7 @@ export function PipelineParametersExecutionDetails(props: IExecutionDetailsSecti
               .map(key => (
                 <React.Fragment key={key}>
                   <dt>{key}</dt>
-                  <dd>{parameters[key]}</dd>
+                  <dd>{parameters[key].toString()}</dd>
```

Comment:
We should probably plan for a long-term fix to show the object values instead of '[object Object]'?

Reply:
I agree. Weirdly enough, it has always shown '[object Object]' , so at least we are back to ground zero 😄